packages/components/psammead-leading-story-promo/src/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from 'styled-components';
+import { node } from 'prop-types';
+import { grid } from '@bbc/psammead-styles/detection';
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+
+/**
+ * Add fallbacks
+ */
+
+const LeadingPromoWrapper = styled.div`
+  position: relative;
+
+  @supports (${grid}) {
+    display: grid;
+    grid-template-columns: repeat(8, 1fr);
+    grid-column-gap: ${GEL_SPACING_DBL};
+    grid-column-end: span 8;
+
+    @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+      grid-column-gap: ${GEL_SPACING_DBL};
+    }
+  }
+`;
+
+// For Image positioning
+// Maybe pull lines 43 -> into ImageGridColumns
+// const ImageGridColumns = css`
+//   grid-column: 1 / span 2;
+// `;
+
+const ImageGridItem = styled.div`
+  display: inline-block;
+  vertical-align: top;
+  position: relative;
+
+  @supports (${grid}) {
+    display: block;
+    width: initial;
+
+    @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
```

Comment:
Do we know if this component will be used just above 1008px? If so, I would remove this media query to keep the grid styles by default.

Otherwise, we might need to tweak it differently under 1008px, rather than just using `display:block`.

Reply:
Mentioned on Slack - ```The styling will be applied only at 1008+ as below that breakpoint we will be maintaining the one column list approach such that all leading promos within a section will look like a top story promo.```